tests/functional/test_new_resolver.py

```diff
@@ -52,7 +52,7 @@ def test_new_resolver_can_install(script):
         "0.1.0",
     )
     script.pip(
-        "install", "--unstable-feature=resolver",
+        "install", "--use-feature=2020-resolver",
```

Comment:
We really ought to have found a better way of doing this, so we don't have to change *every* invocation at each change of stage in the rollout.

Of course, this is only temporary - once the new resolver is released, these tests will just be "normal" tests with no special flag, so it's not worth worrying about.

Reply:
#8430 would improve this significantly -- although I do hear what you're saying.

I'd liked to come up with something general, but time constraints didn't help. :)

Reply:
I must have missed #8430, I don't recall ever looking at it 🙁 

> I'd liked to come up with something general, but time constraints didn't help

It was me that put the initial batch of those tests in there, so if anyone gets the blame it's me! As I say, it's a minor point either way.